css-flexbox/justify-content.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - justify-content</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      #card-deck {
+        display: flex;
+        flex-basis: 25%;
```

Comment:
`flex-basis` is a child property, which means it should be applied to flex children and not the flex container itself. In this case, it should be getting applied to `.card-wrapper`

Reply:
Fixed the following code! thank you!